Client/src/pages/homePage/components/SectionHeroBanner.jsx

```
@@ -11,30 +11,43 @@ const SectionHeroBanner = () => {
+            className="mySwiper rounded-[5px] flex-[7] max-w-[1000px]"
+        >
+            {initialBannerData &&
+                initialBannerData.map((banner, index) => (
+                    <SwiperSlide key={index}>
+                        <img
+                            className="object-cover w-full cursor-pointer rounded-[5px] h-[300px] max-md:h-[200px]"
+                            src={banner?.url_img}
+                            alt={banner?.title || "Banner"}
+                        />
+                    </SwiperSlide>
+                ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)